src/ddmd/intrange.d

```diff
@@ -77,35 +78,70 @@ struct SignExtendedNumber
             return 0;
     }
 
-    SignExtendedNumber opNeg() const
+    private alias SignExtendedNumber R;
```

Comment:
This is a distraction, a future PR should eliminate it and rename `SignExtendedNumber` to `SignExtended`.